lib/elixir/test/erlang/tokenizer_test.erl

```diff
@@ -147,12 +147,12 @@ space_test() ->
   [{op_identifier, {1,1,4}, foo}, {dual_op, {1,6,7}, '-'}, {number, {1,7,8}, 2}] = tokenize("foo  -2").
 
 chars_test() ->
-  [{number, {1,1,3}, 97}]      = tokenize("?a"),
-  [{number, {1,1,3}, 99}]      = tokenize("?c"),
-  [{number, {1,1,4}, 0}]       = tokenize("?\\0"),
-  [{number, {1,1,4}, 7}]       = tokenize("?\\a"),
-  [{number, {1,1,4}, 10}]      = tokenize("?\\n"),
-  [{number, {1,1,4}, 92}]      = tokenize("?\\\\").
+  [{char, {1,1,3}, 97}]      = tokenize("?a"),
```

Comment:
I did that, then `push --force`'d over it and reinserted the whitespace because I expected you to say "this doesn't belong to this commit" :smile: Will do in a moment!